src/utils/Logger.ts

```diff
@@ -0,0 +1,32 @@
+import { LogInterface } from "../interfaces/LogInterface";
+import MessageType from "../types/MessageType";
+
+export class Logger implements LogInterface {
+  public doActive: boolean;
```

Comment:
We don't need the `public` modifier here as JavaScript/TypeScript classes are public by default